src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] resultW = new String[0];
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            String value = reader.readLine();
+            if (value == null) {
+                return resultW;
+            }
+            reader.read();
+            while (value != null) {
```

Comment:
it's duplication checks, think how we can remove them and leave only while the condition